src/firefly/js/ui/PopupPanel.jsx

```diff
@@ -100,16 +100,17 @@ PopupPanel.propTypes= {
     modal : bool,
     visible : bool,
     style : object,
+    sx: object,
     initLeft: number,
     initTop: number,
     onMove: func,
     element,
 };
 
-function PopupHeaderTop({modal,zIndex,left,top,visibility,ctxRef,dialogMoveStart,dialogMoveEnd,
+function PopupHeaderTop({modal,zIndex,left,top,visibility,ctxRef,dialogMoveStart,dialogMoveEnd,sx,
                             onMouseEnter,onMouseLeave,dialogMove,children,title,askParentToClose}) {
     return (
-        <Box style={{zIndex, position:'relative'}}>
+        <Box sx={sx} style={{zIndex, position:'relative'}}>
```

Comment:
@robyww this was bothering me for a while so I decided to look into it. Leaving this comment here for when you look at this on Monday. I think the reason we _need_ this here is because: 

- sx is only applied to MUI/JoyUI components directly 
- so while we're calling PopupPanel via sx here:
  - `<PopupPanel title={title} sx={sx}> {makeContent(content,sx)} </PopupPanel>`
  - I don't think that'll do anything until we apply that `sx` to a JoyUI component 
- so applying it to the `Box` here ensures that the `children` (in this case the `Stack` with `className:FF-Popup-Content` gets our change of `{'.FF-Popup-Content': {width:'40em',maxWidth:'100%'}`
- In fact, I think I can make it cleaner by: 
  - `{'.FF-Popup-Content': {maxWidth:'100%'}, '.ff-PopupPanel':{width:'40em'}}`
  - since this ensures that the parent Card component increases its width, and the child (`DialogContent`) shows all of the content by having its `maxWidth:100%`

Reply:
oh, your right! I will approve the PR. Just make the change so we don’t use style and `sx`
```js
<Box sx={{…sx, zIndex, position:'relative'}}>
```